scripts/docker/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM ubuntu:18.04
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# Basic dependencies
+ENV HOME /opt
```

Comment:
I would leave ENV var settings for the end of the Dockerfile, so that when you decide to change them you don't have to redownload all of your dependencies each time. A good rule of thump is that you want order steps by their likelihood to change, so you can better take advantage of the docker cache

Reply:
I use $HOME throughout the script so I put this one at the top